database/src/models.rs

```diff
@@ -0,0 +1,18 @@
+
+
+
+#[derive(Debug, sqlx::FromRow)]
+pub struct RemoteJob {
```

Comment:
We should rename these struct fields so they are not dependent on the db schema. This will require altering the sql query in the data_accessor function to alias the returned columns with more descriptive names.